### PR TITLE
Bugfix/rest api organizations verify db operation outcome

### LINKED ISSUE #3057: REST API: (organizations) verify DB operation outcome before sending response
## Description

When DB collection content is altered during method call, it must be checked, if the operation was successful before sending response message.

In case operation failed, HTTP code 5xx is used in response message.

At least following methods are concerned:
  * POST /organizations
    * check insert + findOne (if not found, error)  
  * PUT /organizations/{id}
    * check update + findOne (if not found, error)  
  * DELETE /organizations/{id}
    * check removeOrganization 
  * GET /organizations/{id}/managers
    * check users find (looking for managers' user account)
  * POST /organizations/{id}/managers
    * check update (find after update check is OK)
  * GET /organizations/{id}/managers/{managerId}
    * check users find (looking for manager's user account)
  * DELETE /organizations/{id}/managers/{managerId}
    * check removeOrganizationManager

### PR DESCRIPTION
**Pull request by deligence-dharmendra**
Closes #3057

## Changes

- change in organization api.js file
- change in the organization server method.js file

## Developer checklist
This checklist is to be completed by the PR developer:

- [ ] Alternative solutions were compared/discussed before writing code
    - [ ] trade-offs with this solution are considered acceptable
- [ ] Code in this PR adheres to the [project styleguide](CONTRIBUTING.md)
- [ ] This pull request does not decrease project test coverage
- [ ] If the code changes existing database collection(s), migration has been written
- [ ] If UI texts are added or changed, all texts are internationalized

## Reviewer checklist
Reviewed by: @matleppa @marla-singer 

This list is to be completed by the pull request reviewer:
- [ ] Code works as described/expected
- [ ] Code seems to be error free
    - [ ] no browser console errors visible
    - [ ] no server console errors visible
    - [ ] passes CI build
- [ ] Code is written in a way that promotes maintainability
    - [ ] easy to understand
    - [ ] well organized
    - [ ] follows project coding standards and conventions
    - [ ] well documented

### CODE REVIEW

**Comment by marla-singer** [comment]
@matleppa Can you look at this PR?

**Comment by matleppa** [comment]
@marla-singer Yes, I will.
@deligence-dharmendra Please set the "Ready for Review" label, when PR is ready for review.

File: apinf_packages/organizations/server/api.js [hunk 1]
```diff
@@ -914,7 +941,13 @@ ManagementV1.addRoute('organizations/:id/managers', {
       }
 
       // Update Organization manager list
-      Organizations.update(organizationId, { $push: { managerIds: newManager._id } });
+      const result = Organizations.update(organizationId,
+        { $push: { managerIds: newManager._id } });
+
+      // If organiztions update failed
```
Comment (matleppa) [inline apinf_packages/organizations/server/api.js]: organiztions  -->  organizations

File: apinf_packages/organizations/server/api.js [hunk 2]
```diff
@@ -914,7 +941,13 @@ ManagementV1.addRoute('organizations/:id/managers', {
...
+      if (result === 0) {
+        return errorMessagePayload(500, 'Organization update failed');
```
Comment (matleppa) [inline apinf_packages/organizations/server/api.js]: Organization update failed 
-->
Organization manager update failed

**Pull request merged**